test/integration/features/config.feature

```diff
@@ -37,6 +37,18 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
             | nameserver       | 120.0.0.1 |   999.999.999.999 |
             | pull-secret-file |    /Users | /nonexistent-file |
 
+   @linux
+   Scenario: Check bundle version validation (command line)
+       Given executing "touch crc_libvirt_99.9.9.crcbundle" succeeds
+       When executing "crc start --bundle crc_libvirt_99.9.9.crcbundle" fails
```

Comment:
What happens here if binary has bundle embedded in it? Fails with the same message? Can you actually provide an external bundle to such a binary at all (even if correct version)?